templates/base.html

```diff
@@ -51,6 +51,7 @@
                 <li><a href="{% url 'resourcecenter_index' %}">Ressurser</a></li>
                 <li><a href="/wiki/">Wiki</a></li>
                 <li><a href="/webshop/">Webshop</a></li>
+                <li><a href="/hobbygroups/" class="hidden-sm">Interessegrupper</a></li>
```

Comment:
Use the `{% url %}` syntax from Django to generate the URL to the [index page](https://github.com/dotkom/onlineweb4/pull/1847/files#diff-a92964c7df7035eb70f7fffb04eb5615) of the app. (Tip: Look at how it's done for resourcecenter 3 lines above 😉 )